notebooks/DemoSimpleInterface.ipynb

```diff
@@ -108,8 +89,10 @@
     "from torch.autograd import Variable\n",
     "from torch.nn import functional as F\n",
     "\n",
-    "from utils import *\n",
-    "from sklearn.preprocessing import LabelEncoder\n"
+    "from markermap.vae_models import MarkerMap\n",
+    "from markermap.utils import get_zeisel\n",
```

Comment:
what is get_zeisel? Is it from our package? I wonder if it should be something like MarkerMap.get_zeisel if possible.

Reply:
Yeah, `get_zeisel` is one of 4 data import and preprocessing functions that I wrote just because I was writing the same code over and over again. The others are `get_citeseq`, `get_mouse_brain`, and `get_paul`. I will change this import to make it more clear when you are using the function.